packages/components/src/components/Table/Table.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import type { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { Table, THead, TBody, Tr, Th, Td } from "./index";
+
+export default {
+  component: Table,
+  title: "Components/Table",
+} as ComponentMeta<typeof Table>;
+
+const Template: ComponentStory<typeof Table> = () => {
+  return (
+    <Table tableLayout="fixed">
+      <THead>
+        <Tr>
+          <Th>Talent name</Th>
+          <Th>Email</Th>
+          <Th>Current state</Th>
+          <Th>Case type</Th>
+        </Tr>
+      </THead>
+      <TBody>
+        <Tr>
+          <Td>Dietrich Eldert</Td>
+          <Td>dietr-ehler@arketmay.com</Td>
+          <Td>Closed</Td>
+          <Td>Full relocation</Td>
+        </Tr>
+      </TBody>
+    </Table>
+  );
+};
+
+export const Default = Template.bind({});
```

Comment:
Thoughts on adding an example with React Table integrated?

Reply:
Yeah. I can drop a story in for that.

Reply:
@RohitRajendran example story has been added.